src/changelog.ts

```diff
@@ -164,24 +164,19 @@ function getTagUrl(repoUrl: string, tag: string) {
  * @param repoUrl - The URL for the GitHub repository.
  * @param tagPrefix - The prefix used in tags before the version number.
  * @param releases - The releases to generate link definitions for.
+ * @param versionBeforePkgRename - A version string of the package before being renamed.
+ * An optional, which is required only in case of package renamed.
+ * @param tagPrefixBeforePkgRename - A tag prefix string of the package before being renamed.
+ * An optional, which is required only in case of package renamed.
```

Comment:
```suggestion
 * @param [versionBeforePkgRename] - A version string of the package before being renamed.
 * An optional, which is required only in case of package renamed.
 * @param [tagPrefixBeforePkgRename] - A tag prefix string of the package before being renamed.
 * An optional, which is required only in case of package renamed.
```